flink-table/flink-table-common/src/main/java/org/apache/flink/table/sources/LookupableTableSource.java

```diff
@@ -34,12 +34,29 @@
 
 	/**
 	 * Gets the {@link TableFunction} which supports lookup one key at a time.
+	 *
 	 * @param lookupKeys the chosen field names as lookup keys, it is in the defined order
+	 *
+	 * <p>IMPORTANT:
+	 * If the returned {@link TableFunction} receives a lookup request with null value in lookup keys, expect
```

Comment:
Give more detailed comments here maybe more clearly for understanding, 
1. lookupKeys are nullable
2. should carefully deal with `null` value lookup in each concrete store engine, e.g., in MySQL this can a `is null` query
...

Reply:
OK, I'll try to explain more clearly in the comment